package.json

```diff
@@ -5,7 +5,7 @@
     "emotion": "^7.3.2",
     "emotion-server": "^7.3.2",
     "gh-pages": "^1.0.0",
-    "next": "^4.0.0-beta.2",
+    "next": "^4.0.0",
```

Comment:
would you mind updating this to the latest version. There was a more recent version at the time this PR is open. And there was just a newer version released 6 hours ago :)